vaccine/healthcheck_ussd.py

```diff
@@ -892,6 +892,61 @@ async def state_display_risk(self):
                     "TESTING. Go to a testing center or Call 0800029999 or your "
                     "healthcare practitioner for info."
                 )
+
+        if config.TB_USSD_CODE and risk != "high":
+            return MenuState(
+                self,
+                question=self._(text),
+                error=self._(text),
+                choices=[Choice("state_tb_prompt_1", self._("Next"))],
+            )
+        else:
+            return EndState(self, text, next=self.START_STATE)
+
+    async def state_tb_prompt_1(self):
+        answers = self.user.answers
+        risk = self.calculate_risk()
+
+        text = ""
+        if risk == "moderate":
+            if answers.get("state_cough") == "yes":
+                text = self._(
+                    "A cough may also be a sign of TB – a dangerous but treatable "
+                    "disease."
+                )
+            elif risk == "moderate" and answers.get("state_fever") == "yes":
```

Comment:
We already know that `risk == "moderate"` due to the parent if statement

Reply:
ah yes, the parent statement came later and I forgot to remove it